src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -233,9 +232,9 @@ public Token call() throws SaltStackException {
     }
 
     /**
-     * Query for details (grains) of the specified minion
-     *
-     * GET /minions/<minion-id>
+     * Query for details (grains) of the specified minion.
+     * <p>
```

Comment:
It is, the point seems to be just to not blow up the source code too much, see here:
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format 
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#examples